README.md

```diff
@@ -27,11 +27,11 @@ cluster on IBM Cloud.
 
 This provider's versions are compatible with the following versions of Cluster API:
 
-|                                            | v1alpha4 (v0.4) | v1beta1 (v1.x) |
-|--------------------------------------------|-----------------|----------------|
-| IBMCloud Provider v1alpha4 (v0.1.x)        | ✓               |                |
-| IBMCloud Provider v1beta1 (v0.2.x, v0.3.x) |                 | ✓              |
-| IBMCloud Provider v1beta2 (v0.4.x, v0.5.x, v0.6.x, main) |                 | ✓              |
+|                                         |Cluster API v1alpha4 (v0.4) |Cluster API v1beta1 (v1.x) |
+|:----------------------------------------|:---------------:|:--------------:|
+| CAPI v1alpha4 (v0.1.x)                  | ✓               |                |
```

Comment:
why is this CAPI? this is suppose to be CAPIBM

Reply:
@mkumatag, I have made the required changes. 
Thanks!

Reply:
@mkumatag, a miss from my end. Will update accordingly. 
Thanks!